src/modules/wireplumber.cpp

```diff
@@ -17,32 +17,26 @@ waybar::modules::Wireplumber::Wireplumber(const std::string& id, const Json::Val
       volume_(0.0),
       min_step_(0.0),
       node_id_(0) {
+  spdlog::set_level(spdlog::level::trace);
```

Comment:
i dont think we need this anymore? assuming it was for development

Reply:
Thanks, removed.